docs/content/getting-started/_index.md

```diff
@@ -92,17 +125,155 @@ class App extends Component {
 
 ## 4. Add attributes 
 
-You may choose to add attributes such as `priority` or `color` as needed to adjust usage of palette color, context, or priority.
+You may choose to add attributes such as `pfe-variant`, `pfe-priority` or `pfe-color` as needed to adjust usage of general theme, palette color, or priority. Check the readme file of a component to see what attributes are supported. Or check out the [Storybook](https://patternfly.github.io/patternfly-elements/demo) to preview how the attributes work.
 
-```html
-  <pfe-card color="darkest">
-    <h2 slot="header">Dark card</h2>
-    <p>Lorem ipsum.</p>
-    <pfe-cta color="complement" priority="primary" on="dark" slot="footer">
-        <a href="#">Important call-to-action</a>
-    </pfe-cta>
-  </pfe-card>
+#### General
+
+  Priority & variant attributes change style styles globally:
```

Comment:
I think this should read:
Priority & variant attributes change styles globally:

Reply:
thanks, updated